sdk/eventhub/event-hubs/package.json

```diff
@@ -106,7 +106,7 @@
   "devDependencies": {
     "@azure/dev-tool": "^1.0.0",
     "@azure/eslint-plugin-azure-sdk": "^3.0.0",
-    "@azure/identity": "^1.1.0",
+    "@azure/identity": "2.0.0-beta.3",
```

Comment:
is this intentionally beta.3 vs beta.4?

Reply:
No I was in the middle of conversation with Daniel when I made this beta.3, then we decided to go with beta.4. Thanks!